### PR TITLE
PMM-7190 notification channels UI tests

### PR DESCRIPTION
**Pull request by yurkovychv**
[PMM-7190](https://jira.percona.com/browse/PMM-7190)

### CODE REVIEW

**Review by lunaticusgreen — DISMISSED** [review]
👍

File: pmm-app/tests/ia/notificationChannels_test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+for (const [, channel] of Object.entries(page.types)) {
+  notificationChannels.add([channel.name, channel.type]);
+}
+
+Feature('IA: Notification Channels');
+
+
+Before(async (I, channelsAPI) => {
+  I.Authorize();
+  await channelsAPI.clearAllNCs();
```
Comment (atymchuk) [inline pmm-app/tests/ia/notificationChannels_test.js:14]: nit: hard to read without the context, `clearAllNotificationChannels` would read better :)

File: pmm-app/tests/ia/pages/api/channelsAPI.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    if (resp.data === {}) return;
+
+    for (const i in resp.data.channels) {
+      const channel = resp.data.channels[i];
+
+      await this.deleteNC(channel.channel_id);
+    }
+  },
+
+  async deleteNC(channelId) {
```
Comment (atymchuk) [inline pmm-app/tests/ia/pages/api/channelsAPI.js:58]: nit: Likewise, `deleteNotificationChannel` would read better

File: pmm-app/tests/ia/pages/notificationChannelsPage.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        I.fillField(this.fields.emailsInput, this.types.email.addresses);
+        break;
+      case this.types.pagerDuty.type:
+        I.fillField(this.fields.routingKeyInput, this.types.pagerDuty.key);
+        break;
+      case this.types.slack.type:
+        I.fillField(this.fields.slackChannelInput, this.types.slack.slackChannel);
+        break;
+      default:
+        assert.ok(false, `Did not found matching notification channel type ${type}`);
```
Comment (atymchuk) [inline pmm-app/tests/ia/pages/notificationChannelsPage.js:98]:
```suggestion
        assert.ok(false, `Did not find a matching notification channel type ${type}`);
```

File: pmm-app/tests/ia/pages/notificationChannelsPage.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        I.appendField(this.fields.emailsInput, suffix);
+        break;
+      case this.types.pagerDuty.type:
+        I.fillField(this.fields.routingKeyInput, suffix);
+        break;
+      case this.types.slack.type:
+        I.fillField(this.fields.slackChannelInput, suffix);
+        break;
+      default:
+        assert.ok(false, `Did not found matching notification channel type ${type}`);
```
Comment (atymchuk) [inline pmm-app/tests/ia/pages/notificationChannelsPage.js:121]:
```suggestion
        assert.ok(false, `Did not find a matching notification channel type ${type}`);
```

**Review by atymchuk — APPROVED** [review]

**Review by puneet0191 — APPROVED** [review]

File: pmm-app/tests/ia/notificationChannels_test.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/ia/notificationChannels_test.js:14]: renamed, thanks

File: pmm-app/tests/ia/pages/api/channelsAPI.js (hunk 2 above)
Comment (yurkovychv) [reply pmm-app/tests/ia/pages/api/channelsAPI.js:58]: renamed, thanks 😊

**Review by atymchuk — APPROVED** [review]

**Pull request merged**